okapi-core/src/main/java/org/folio/okapi/service/impl/ProcessModuleHandle.java

```diff
@@ -149,7 +149,7 @@ private NuProcess launch(EnvEntry[] env, String [] command) {
     Promise<Void> promise = Promise.promise();
     // time to wait for process status.. when a port is present (always in real life)..
     // The waitReady will check if process eventually starts listening on port
-    vertx.setTimer(port == 0 ? 3000 : 1000, timerRes -> {
+    vertx.setTimer(3000, timerRes -> {
```

Comment:
How do you conclude that the NuPocess is not returning the right process? As far as I can tell it reaches waitReady as it should, but the Test doesn't wait enough time.. So I think we should just increase waitIterations in ProcessModuleHandleTest, say, from 15 to 20 (desc.setWaitIterations(15)).

The test runs a Bourne Shell script and that's the exit code we check (also on Linux).

Increasing from 1 to 3 seconds will make ALL process deployments 2 seconds slower.

Reply:
I tried that initially and it did not work. I added some debug statements and found out the PID difference. For example:
```
Starting test: testCmdlineOk
07:59:13 [] [] [] [] INFO  Messages             Loading messages from /infra-messages/Messages_en.properties
07:59:15 [] [] [] [] INFO  TcpPortWaiting       tryConnect() host localhost port 9231 count 0
res suceeded: false
process is null? false
process id 52222
process is running? false
```
The actual Java process
```
hji              52223   0.0  0.6 10281380  98932   ??  S     7:59AM   0:02.48 /usr/bin/java -DpidFile=test-module.pid -Dport=9231 -jar ../okapi-test-module/target/okapi-test-module-fat.jar
```

Reply:
The PID is a red herring..  Not related.. The PID is the Bourne Shell on my system too. The error message is coming from the code that waits for the port to become ready.. Please increase it to 20.

Reply:
It does not work. See below console output for failed testCmdlineOk test. The reason it does not work is here https://github.com/folio-org/okapi/blob/master/okapi-core/src/main/java/org/folio/okapi/util/TcpPortWaiting.java#L41-L53. 
When res is false, proces is not null and not running (because of PID mismatch), the try maxIteration loop does not kick in. BTW, in AWS ec2 linux evn, the PID from sh and Java is the same.  But on my macOS that is not the case.
```
Starting test: testCmdlineOk
09:39:23 [] [] [] [] INFO  Messages             Loading messages from /infra-messages/Messages_en.properties
09:39:25 [] [] [] [] INFO  TcpPortWaiting       tryConnect() host localhost port 9231 count 0
maxIterations is 20
res suceeded: false
process is null? false
process id 54052
process is running? false
```

Reply:
You are right .. isRunning is the problem .. So we should not even bother to check it when the process spawned is a Bourne shell .. (cmdLineStart, cmdLineStop).

Reply:
Made changes suggested by @adamdickmeiss